project_task_stage_allow_timesheet/views/project_task_type.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="project_task_type_form_view">
```

Comment:
just cosmetical change (optional):
id before model
and use of original id (task_type_edit)